### DISCUSSION TITLE
Unable to access my github page with custom domain

### DISCUSSION THREAD

**Discussion by skmdab** [community/community#111130] [Other Feature Feedback, Questions, & Ideas]
### Github pages

General

### Body

Hi,

I have created github page and added custom domain to it and also added A records github ip addresses in my cloudflare. But i am not able to access my github page with my custom domain.

Help will be appreciated

**Comment by queenofcorgis** [comment]
Hi there @skmdab 👋🏼 

Are you getting an error message, if you could share some more details you may be more likely to get a helpful response 😃

**Reply by skmdab** [reply]
Thanks for replying. Like Any Error message i didn't seen there. But Issue was resolved.

**Reply by Zakir-Muhammad** [reply]
@skmdab How was the issue resolved? I'm having a similar issue.

**Reply by skmdab** [reply]
> @skmdab How was the issue resolved? I'm having a similar issue.

I am using cloudflare pages instead of github pages.
